spec/factories.rb

```diff
@@ -1,3 +1,5 @@
+Monban.test_mode!
```

Comment:
@Fatpaher Why this line here? I don't think that this is related to factories. Move it to `rails_helper.rb` please.